.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "module/display_drivers"]
+	path = module/display_drivers
+	url = git@github.com:engdoreis/display_drivers.git
```

Comment:
Do I have access to do so? Or maybe @GregAC could fork the repo.

Reply:
Yeah I can bring it in as a lowRISC repo.

Though I'd like to vendor it in rather than use git submodules to match our approach for other things in the demo system

Reply:
Done